### ISSUE TITLE
[🐞]: Moving Treasure Maps to inventory causing MC to crash (1.20.1)

### ISSUE THREAD

**Issue by JonnyM5** [MehVahdJukaar/Moonlight#179]
### BEFORE CONTINUING:

- [X] Always check you are using the latest version of these mods and its dependencies
- [X] Remove known problematic rendering mods such as Optifine, Sodium and others

### Version-Loader

1.20.1-forge

### Moonlight Lib Version

moonlight-1.20-2.8.67

### Issue Detail

MC is crashing whenever the client is taking a treasure map and putting it inside their inventory. After that MC is directly crashing...
Supplementaries is installed, but I could not figure out if that is the problem.

### Log Attachment

[crash-2023-12-15_16.26.48-client.txt](https://github.com/MehVahdJukaar/Moonlight/files/13687419/crash-2023-12-15_16.26.48-client.txt)

### OPTIONAL: To Produce

1) join server
2) open chest
3) take treasure map and move to inventory
4) crash

**Comment by MehVahdJukaar** [comment]
Did you actually do step 2 or just ticked the box without reading it?

**Comment by JonnyM5** [comment]
I did the test! Dont worry. :D
And the issues seems to be just on the server, in single player I cant recreate the issue. (2 people got the problem at this point)

**Comment by JonnyM5** [comment]
I also tried older versions of the mod, but unfortunately I got nowhere. Hope you might be able to have a fix? 👍

**Comment by MehVahdJukaar** [comment]
Then send the log from when you are not using otpifine pls

**Comment by JonnyM5** [comment]
Hey, in the last hour we tried many things and we have no idea, what we actually did... I am so sorry for taking your time. We tried many things beforehand, I would never write an issue, if we did not try many things. 
Right now everything seems to be normal and we actually can pick up treasure maps. 

Thank you for your help in this matter, we wish you well and a great christmas.
Sorry again for taking your time!
